System.IO.Streams/StreamWriter.cs

```diff
@@ -51,9 +57,10 @@ public override string NewLine
         /// Initializes a new instance of the <see cref="StreamWriter"/> class for the specified stream by using UTF-8 encoding and the default buffer size.
         /// </summary>
         /// <param name="stream">The stream to write to.</param>
+        /// <param name="leaveOpen">Leave stream open after dispose writer.</param>
```

Comment:
```suggestion
        /// <param name="leaveOpen"><see langword="true"/> to leave the stream open after the <see cref="StreamWriter"/> object is disposed; otherwise, <see langword="false"/>.</param>
```